src/main/java/edu/wpi/first/pathweaver/CreateProjectController.java

```diff
@@ -75,16 +75,18 @@ public Game fromString(String string) {
 
   @FXML
   private void createProject() {
-    String folder = directory.getText().trim();
-    ProgramPreferences.getInstance().addProject(folder);
+    String folderString = directory.getText().trim();
+    File directory = new File(folderString, "PathWeaver");
+    directory.mkdir();
```

Comment:
I'd do `mkdirs` here to make sure the parent directories are created if they don't already exist

Reply:
So because this folderString comes from a DirectoryChooser I think we should be good for the parent directories.

Reply:
Then wouldn't the directory also exist? I don't think directory choosers can select nonexistent directories

Reply:
No, because we are opening a sub-directory "PathWeaver" of the chosen directory, and that most likely will not be existent.